contrib/internal/httptrace/options_test.go

```diff
@@ -0,0 +1,63 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2022 Datadog, Inc.
+
+package httptrace
+
+import (
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestConfig(t *testing.T) {
+	t.Run("client-ip", func(t *testing.T) {
+		t.Run("unset", func(t *testing.T) {
+			restore := cleanEnv()
+			err := os.Unsetenv(clientIPHeaderEnvVar)
+			require.NoError(t, err)
+			cfg := newConfig()
+			require.Empty(t, cfg.ipHeader)
+			defer restore()
+
+		})
+
+		t.Run("empty", func(t *testing.T) {
+			restore := cleanEnv()
+			err := os.Setenv(clientIPHeaderEnvVar, "")
+			require.NoError(t, err)
+			cfg := newConfig()
+			require.Empty(t, cfg.ipHeader)
+			defer restore()
+
+		})
+
+		t.Run("set", func(t *testing.T) {
+			restore := cleanEnv()
```

Comment:
Why not
```suggestion
			defer cleanEnv()()
```

Reply:
Fine by me, just wasn't sure if this is okay in in general. Those tests should disappear for now though since _options.go_/_options_test.go_ will be discarded and the user ip header behaviour from `getClientIP` is already tested in _httptrace_test.go_